### ISSUE TITLE
Low quality HTML and CSS

### ISSUE THREAD

**Issue by naomiaz** [azure-ad-b2c/html-templates#19]
I'm trying to adjust the branding of the templates for a client, but it is an extreme mess. The HTML contains too many generic classes, and using ID selectors in CSS is bad practice in general because of specificity issues. 
Will this get fixed in the future?

**Comment by rovangju** [comment]
Agreed - I'm also confused as to why there's a boat load of styles that are imported by gulp, but we don't get access to those source files?

**Comment by gopal-amlekar** [comment]
I agree, it is a nightmare to fix / rewrite the styles. I used prettier formatter in VSCode and it then shows all the styles (gulp generated) in a readable format. I then inspect each element, find out its class and literally overwrite it in my own css with all the specificity. 
Besides, it uses Bootstrap 3. I tried to use BS 5 but it breaks the styles. BS 4 works but with some changes to the template.
Another scary stuff is the Handlebar generated JS scripts. It makes it very difficult to understand certain behaviours like the form validation.
